rust_src/src/lib.rs

```diff
@@ -85,6 +97,20 @@ pub extern "C" fn rust_init_syms() {
         defsubr(&*character::Smax_char);
 
         floatfns::init_float_syms();
+
+        // The largest value that is representable in a Lisp integer
+        defvar!(b"most-positive-fixnum", f_Vmost_positive_fixnum);
+        rust_make_symbol_constant(intern_c_string_1(b"most-positive-fixnum"
```

Comment:
yes, `\0` needs to be added, but by the `defvar!` caller. I going to fix it and rebase.